opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -22,6 +22,7 @@
 from typing import Optional
 
 from opentelemetry.context import Context, attach, detach, set_value
+from opentelemetry.instrumentation.utils import _SUPPRESS_INSTRUMENTATION_KEY
```

Comment:
This will require an update to the SDK's setup.cfg to add a dependency on the `opentelemetry-instrumentation` package.

Reply:
I just have a quick follow up question regarding this. When I add `opentelemetry-instrumentation == 0.23.dev0` to the setup.cfg, I get the following error:

`No matching distribution found for opentelemetry-instrumentation==0.23.dev0` 

However, I do see that the opentelemetry-instrumentation package version is 0.23.dev0 and assume I'm missing something in the setup.cfg file. Please let me know if you have any tips on fixing this issue! Thanks 😊

Reply:
yeah it's because pip is searching pypi for a version of `opentelemetry-instrumentation` and not finding the dev version. you'll need to update tox.ini to install the instrumentation package from the local copy, i think here https://github.com/open-telemetry/opentelemetry-python/blob/ea4cdfe42435dda7c39a264612e89dae09eac620/tox.ini#L121

you can take a look at the rest of tox.ini for examples of how to install opentelemetry-instrumentation